taigit/src/libraries/Taiga.ts

```diff
@@ -147,6 +147,36 @@ task_history(taskId : number) : Promise<Object> {
     return output;
 }
 
+/**
+ * @summary Get the history of an userstory 
+ * @param userstoryId the ID for the userstory to fetch the history
+ * @returns userstory history data object
+ */
+export async function
+us_history(userstoryId : number) : Promise<Object> {
+    let data = (await axios.get("https://api.taiga.io/api/v1/history/userstory/" + userstoryId.toString())).data;
+    //https://api.taiga.io/api/v1/history/userstory/2698838
+
+    let output : Array<Object> = [];
+    for(let entry of data) {
+        let new_entry = {
+            user_id : entry.user.pk,
+            user_name : entry.user.username,
+            name : entry.user.name,
+            user_status : entry.user.is_active,
+            timestamp : new Date(entry.created_at),
+            us_key : entry.key,
+            us_diff : Object.keys(entry.diff),
+            us_values : Object.keys(entry.values),
+            us_diff_values : Object.keys(entry.values_diff),
+            comment : entry.comment   
+        }
```

Comment:
You should create a type interface for US history objects. will make this a lot simpler to declare and iterate.